src/components/messenger/list/conversation-list-panel/index.tsx

```diff
@@ -68,32 +68,46 @@ export class ConversationListPanel extends React.Component<Properties, State> {
     this.setState({ userSearchResults: filteredItems?.map(itemToOption) });
   };
 
+  // Checks if the conversation's name matches the search filter.
```

Comment:
I feel like maybe there's some improvement that could be made here. Generally speaking, if you have to describe what the function is doing with a comment then that means the code isn't clear enough (not a global rule...but a reasonable guideline). Can we come up with some better names or more refactoring to clear this up?

Reply:
Hi Dale. Had a go at some refactoring here. Decided to move helpers into a utils file to keep the conversation list panel code a bit cleaner. Also updated the names of the helpers to improve readability and thus remove the need for comments. Let me know what you think of the updates. Thanks!